src/main/java/com/lineate/mdzinarishvili/reminderapp/dao/TokenDaoImpl.java

```diff
@@ -29,20 +32,24 @@ public TokenDaoImpl(JdbcTemplate jdbcTemplate) {
 
 
   public List<Token> findAllValidTokenByUser(Long id) {
+    log.info("Getting all valid tokens by user with id " + id);
```

Comment:
you should use logger placeholders. Like that
log.info("Getting all valid tokens by user with id={}", id);
you can use a few placeholders in one string
log.info("First var={}, second var={}", firstVar, secondVar);